duecredit/injections/mod_sklearn.py

```diff
@@ -32,3 +32,45 @@ def inject(injector):
         publisher={JMLR. org}
         }
     """), description="Machine Learning library")
+
+    # sklearn.cluster.affinity_propagation_
+    injector.add('sklearn.cluster.affinity_propagation_', None, Doi('10.1126/science.1136800'),
+                 description="Affinity propagation clustering algorithm", tags=['reference'])
+
+    # sklearn.cluster.bicluster
+    injector.add('sklearn.cluster.bicluster', 'SpectralCoclustering', Doi('10.1.1.140.301'),
+                 description="Spectral Coclustering algorithm", tags=['reference'])
+    injector.add('sklearn.cluster.bicluster', 'SpectralBiclustering', Doi('10.1.1.135.1608'),
+                 description="Spectral Biclustering algorithm", tags=['reference'])
```

Comment:
I just read the documentation :-) which says

```
          - "implementation" [default] tag describes as an implementation of the cited
             method
          - "reference" tag describes as the original implementation of
            the cited method
```
and those are all different algorithms, I think.